ToDoOrNotToDo/app/src/main/java/shuvalov/nikita/todoornottodo/MasterLister.java

```diff
@@ -0,0 +1,38 @@
+package shuvalov.nikita.todoornottodo;
+
+import java.util.ArrayList;
+
+/**
+ * Created by NikitaShuvalov on 10/20/16.
+ */
+public class MasterLister {
+    private static ArrayList<ErrandsEncapsulator> mMasterList;
+    private static MasterLister ourInstance = new MasterLister();
+
+    public static MasterLister getInstance() {
+        return ourInstance;
+    }
+
+    public static ArrayList<ErrandsEncapsulator> retriveToDoLists(){
```

Comment:
Only the `getInstance()` method should be static. The other methods should not be static, meaning you have to get _the_ instance before you can call the other methods. The reason for this is that the other methods depend on the constructor having been called so that your `mMasterList` is not null. In your case, you called the constructor as you declared `ourInstance` but if you follow the pattern from the lesson and lazily instantiate `ourInstance` only when it is first needed, then having the other methods be static could cause null pointer exceptions.